packages/app/obojobo-repository/shared/components/dashboard.jsx

```diff
@@ -65,62 +65,113 @@ const renderModalDialog = props => {
 	)
 }
 
-const renderModules = modules => {
-	return modules.map(draft => <Module key={draft.draftId} hasMenu={true} {...draft}></Module>)
-}
+class Dashboard extends React.Component {
+	constructor(props) {
+		super(props)
 
-const Dashboard = props => {
-	return (
-		<span id="dashboard-root">
-			<RepositoryNav
-				userId={props.currentUser.id}
-				avatarUrl={props.currentUser.avatarUrl}
-				displayName={`${props.currentUser.firstName} ${props.currentUser.lastName}`}
-				noticeCount={0}
-			/>
-			<RepositoryBanner title={props.title} className="default-bg" facts={props.facts} />
-			<div className="repository--section-wrapper">
-				<section className="repository--main-content">
-					<div className="repository--main-content--control-bar">
-						<MultiButton title="New Module">
-							<Button
-								onClick={() => {
-									props.createNewModule(false)
-								}}
-							>
-								New Module
-							</Button>
-							<Button
-								onClick={() => {
-									props.createNewModule(true)
-								}}
-							>
-								New Tutorial
-							</Button>
-						</MultiButton>
-						<Search
-							value={props.moduleSearchString}
-							placeholder="Filter..."
-							onChange={props.filterModules}
-						/>
-					</div>
-					<div className="repository--main-content--title">
-						<span>My Modules</span>
-					</div>
-					<div className="repository--item-list--collection">
-						<div className="repository--item-list--collection--item-wrapper">
-							<div className="repository--item-list--row">
-								<div className="repository--item-list--collection--item--multi-wrapper">
-									{renderModules(props.filteredModules ? props.filteredModules : props.myModules)}
+		const selectedSort = props.selectedSort
+
+		this.state = {
+			selectedSort,
+			sortOptions: ['Newest', 'Alphabetical', 'Last updated']
```

Comment:
Don't keep options like these in state - their constants for the 'class'  either create a const in the scope somewhere or make them a property of the class.